bamboo/unit-tests.sh

```diff
@@ -8,5 +8,9 @@ set -ex
 docker ps -a ## Show running containers for output logs
 
 docker exec -i ${container_id}-build_env-1 /bin/bash -c "cd $UNIT_TEST_BUILD_DIR && npm run db:local:reset"
-docker exec -i ${container_id}-build_env-1 /bin/bash -c "cd $UNIT_TEST_BUILD_DIR && npm run test:coverage"
+docker exec -i ${container_id}-build_env-1 /bin/bash -c "cd $UNIT_TEST_BUILD_DIR && npm run test:ci"
+if [ -n "$(ls -A ./unit-logs/@cumulus 2>/dev/null)" ]
```

Comment:
doing it, there was a "I don't think I can do that" only to think of the solution over night

Reply:
just kidding, it doesn't work, I've moved back to having this push in the unit-tests file because I can't get it to run correctly from inside the docker-container. I'm continuing to try to figure this out but have reverted to this form because this is time boxed and I intend to be done with it today

Reply:
Hmm, ok, fair enough, there is a *different* maintenance headache getting bamboo/container/bash to play nice with creds and such.